docs/src/templates/pages/styleguide/ui-templates/content-pages/index.hbs

```diff
@@ -1,23 +1,12 @@
 ---
-title: Component Library
+title: Content Pages
```

Comment:
Think this page can just be deleted – think it's likely a hangover from the old docs (or something else).

Notice it has a link to it in the subnav too, but this can be removed too.

Reply:
I've updated it so that it contains a list of the content pages now, rather than listing them all on the main UI Templates pages. Do you think I should remove these pages and display all of the links on the main page instead?

Reply:
Posted that and read your next comment d'oh!

Reply:
Yeh – I think this will be a good idea if we end up with too many pages on that page eventually that it becomes unmanageable.  But as we've only got these 4 pages atm, I'd just keep them there for now under different titles.